csi.proto

```diff
@@ -475,8 +475,10 @@ message ListVolumesResponse {
     // This token allows you to get the next page of entries for
     // `ListVolumes` request. If the number of entries is larger than
     // `max_entries`, use the `next_token` as a value for the
-    // `starting_token` field in the next `ListVolumes` request. This
-    // field is OPTIONAL.
+    // `starting_token` field in the next `ListVolumes` request. The
+    // Plugin my choose to use the token to contain all the necessary
```

Comment:
s/my/may/

I'm not sure I understand the need for this change. Maybe I'm too close to the spec? My understanding was that the statement added here is implicit since the plugin generates the tokens to begin with.

Reply:
In other words, in what case would a plugin NOT generate a token that's sufficient to serve subsequent requests?

Reply:
I agree with you James. I’m not sure there needs to be clarification. There’s a reason it’s “string next_token” and not “unit64 next_page”. The token is whatever the SP wants it to be and opaque to the client.

Reply:
Yep, I agree that both of you are too close to the spec :smile: 

As a driver developer, this text clears up how to use it.